torchaudio/transforms.py

```diff
@@ -418,6 +426,8 @@ class MelSpectrogram(torch.nn.Module):
             :attr:`center` is ``True``. Default: ``"reflect"``
         onesided (bool, optional): controls whether to return half of results to
             avoid redundancy. Default: ``True``
+        norm (Optional[str]): If 'slaney', divide the triangular mel weights by the width of the mel band
+        (area normalization). (Default: ``None``)
```

Comment:
Hi, thank you for this PR. This line has a small typo that is causing a formatting issue in the docs. 
![image](https://user-images.githubusercontent.com/47190785/110047306-a58cf700-7d1b-11eb-9537-1c809c94979b.png)

Line 430 should be indented so the docs generator doesn't think it's a new argument. Thank you!